test/Microsoft.AspNet.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -143,34 +143,6 @@ public void ParentItems_SetsItemsProperty()
             Assert.Empty(content3.GetContent(new HtmlTestEncoder()));
         }
 
-        [Fact]
-        public async Task ExecuteChildContentAsync_IsNotMemoized()
```

Comment:
It's gone, yes. However, I wouldn't say it was ever user surface. It only existed on the `TagHelperExecutionContext` for the `TagHelperRunner` system to consume.